ci-operator/step-registry/cucushift/installer/check/vsphere/template/cucushift-installer-check-vsphere-template-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+template_config=$(yq-go r "${SHARED_DIR}/install-config.yaml" 'platform.vsphere.failureDomains[*].topology.template')
+template_config=${template_config##*/}
+template_cluster=$(oc get machineset -n openshift-machine-api -ojson | jq -r '.items[].spec.template.spec.providerSpec.value.template')
```

Comment:
Is it possible to use govc command to get what template is being used for vm?

Reply:
if use govc command to get template, can only get rhcos version. unable to get the hardware version

Reply:
okay.

Reply:
seems using `govc events ${vm path}` can check vm deployed from which template.
```
# govc events /DEVQEdatacenter/vm/ci-op-2xd474w5-67746-xbrwf/ci-op-2xd474w5-67746-xbrwf-master-1 | grep "ci-op-2xd474w5-67746-xbrwf-rhcos-generated-region-generated-zone"
[Mon Apr  1 23:47:28 2024] [info] Deploying ci-op-2xd474w5-67746-xbrwf-master-1 on host devqe-vmware-host-2.devqe.ibmc.devcluster.openshift.com in DEVQEdatacenter from template ci-op-2xd474w5-67746-xbrwf-rhcos-generated-region-generated-zone
[Mon Apr  1 23:47:38 2024] [info] Template ci-op-2xd474w5-67746-xbrwf-rhcos-generated-region-generated-zone deployed to ci-op-2xd474w5-67746-xbrwf-master-1 on devqe-vmware-host-2.devqe.ibmc.devcluster.openshift.com,
```